assets/base/i18n/en.json

```diff
@@ -853,6 +859,8 @@
   "share_feedback.title": "Would you share your feedback?",
   "smobile.search.recent_title": "Recent searches in {teamName}",
   "snack.bar.favorited.channel": "This channel was favorited",
+  "snack.bar.channel.member.added": "1 member added",
```

Comment:
use intl `plural` option and remove second id  https://formatjs.io/docs/intl-messageformat/